### ISSUE TITLE
Dice + Cross entropy loss

### ISSUE THREAD

**Issue by wyli** [Project-MONAI/MONAI#1157]
**Is your feature request related to a problem? Please describe.**
Dice + Cross entropy loss is discussed in #807 and implemented in https://github.com/Project-MONAI/tutorials/blob/master/modules/dynunet_tutorial.ipynb
it should be integrated into the main codebase

**Comment by wyli** [comment]
Hi @yiheng-wang-nv, could you please work on this one?

**Comment by Nic-Ma** [comment]
@wyli , we are also discussing how to move the dynunet tutorial into Clara SDK, this ticket can be quite useful.

Thanks.

**Comment by wyli** [comment]
> @wyli , we are also discussing how to move the dynunet tutorial into Clara SDK, this ticket can be quite useful.
> 
> Thanks.

sure thanks, for all the relevant PRs to MONAI we should keep @FabianIsensee in the loop

**Comment by natacanta5** [comment]
Hi, I have a question, the cross-entropy class instead of having to create manually that, the monais project would include this class into its code? Just to easily import it.
